packages/rich-text/src/plugins/DragAndDrop/index.tsx

```diff
@@ -48,7 +49,10 @@ export function createDragAndDropPlugin(): PlatePlugin {
         if (!dropDisallowed) {
           // Move the drop event to a new undo batch mitigating the bug where undo not only moves it back,
           // but also undoes a previous action: https://github.com/ianstormtaylor/slate/issues/4694
-          editor.history.undos.push([]);
+          (editor as unknown as HistoryEditor).history.undos.push({
```

Comment:
slate-history behind the scenes got refactored to use batch objects with operations instead of an array of arrays